metpy/calc/tests/test_thermo.py

```diff
@@ -733,3 +733,20 @@ def test_mixed_layer():
     temperature = np.array([22.2, 14.6, 12., 9.4, 7., -38.]) * units.degC
     mixed_layer_temperature = mixed_layer(pressure, temperature, depth=250 * units.hPa)[0]
     assert_almost_equal(mixed_layer_temperature, 16.4024930 * units.degC, 6)
+
+
+def test_thickness_hydrostatic():
+    """Tests the thickness calculation."""
+    pressure = np.array([959., 779.2, 751.3, 724.3, 700., 269.]) * units.hPa
+    temperature = np.array([22.2, 14.6, 12., 9.4, 7., -38.]) * units.degC
+    mixing = np.array([0.01458, 0.00209, 0.00224, 0.00240, 0.00256, 0.00010])
+    thickness = thickness_hydrostatic(pressure, temperature, mixing=mixing)
+    assert_almost_equal(thickness, 10725.92 * units.m, 2)
+
+
+def test_thickness_hydrostatic_isothermal():
```

Comment:
Travis error is coming from rounding on legacy here:
```python
E           AssertionError: 
E           Arrays are not almost equal to 2 decimals
E            ACTUAL: 5542.1180018995983
E            DESIRED: 5542.11
```

Reply:
Hopefully this should be fixed with the formula and test updates I was just able to make.